CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+
+## API and Backend Dependencies
```

Comment:
I guess these aren't the sort of things I'd normally expect in a CONTRIBUTING doc?  The way Github links to them, they're normally used to communicate rules and processes for contributing to an OSS project.  Not information on installing/setup/running-tests.